services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeController.kt

```diff
@@ -53,8 +76,38 @@ class HankeController(
         return hanke
     }
 
-    @GetMapping
-    fun getHankeList(@RequestParam geometry: Boolean = false): List<Hanke> {
+    @GetMapping(produces = [APPLICATION_JSON_VALUE])
+    @Operation(
+        summary = "Get hanke list",
+        description = "Get list of hanke to which the user has permission to."
```

Comment:
This could use a bit of clarification on how it differs from 'Get public hanke list'.

Reply:
Done.